kbc_transformation/transformation.py

```diff
@@ -84,15 +84,15 @@ def install_packages(packages):
         import sys
         for package in packages:
             args = [
-                sys.executable,
+                os.environ['VIRTUAL_ENV'] + '/bin/python',
```

Comment:
interesting, I would thought the virtual env handles this automagically :thinking: it does not act as chroot?

Reply:
yes, It would work here, but this is required because in the sandbox it is executed in system python, but we want to install the packages to the venv